_tuning-your-cluster/seperate-index-and-search-workloads.md

```diff
@@ -171,3 +173,23 @@ POST my_index/_scale
 }
 ```
 {% include copy-curl.html %}
+
+#### Search replica recovery scenarios
+
+OpenSearch handles recovery of search replicas in search-only mode differently depending on your configuration:
+
+**Scenario 1: Persistent data directory with remote store state disabled**
+
+When you have persistent data and `cluster.remote_store.state.enabled` set to `false`, search replicas recover automatically after node restarts.
+
+**Scenario 2: Remote store state enabled without persistent data directory**
+
+When `cluster.remote_store.state.enabled` is `true` but you don't have a persistent data directory, search replicas recover without primaries and write replicas. Because remote store state is enabled, OpenSearch remembers that the index exists after restart. The allocation logic skips checking for an active primary for search replicas, so search replicas are allocated and search queries remain functional.
+
+**Scenario 3: Remote store state enabled with persistent data directory**
```

Comment:
how is this different than scenario 1? it automatically recovers because you have the data directory, not because of the remote state.  I think we can simplify this section to only the first two scenarios.  or a smaller paragraph

Reply:
With scenario 1 the `cluster.remote_store.state.enabled` is `false` and has persistent data directory. The metadata info is on local node. But with  scenario 3 is when just `cluster.remote_store.state.enabled` set to true with no persistent data directory. With scenario 3 the search replicas will still come up without any manual intervention as it will pull the metadata from remote store and allocates the search replicas.